digitalocean/import_digitalocean_kubernetes_cluster_node_pool_test.go

```diff
@@ -0,0 +1,51 @@
+package digitalocean
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDigitalOceanKubernetesClusterNodePool_Import(t *testing.T) {
```

Comment:
@andrewsomething: This test fails with:

```
$ TF_LOG=DEBUG TF_LOG_PATH=./log.out make testacc TESTARGS='-run=TestAccDigitalOceanKubernetesClusterNodePool_Import'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccDigitalOceanKubernetesClusterNodePool_Import -timeout 120m
?       github.com/terraform-providers/terraform-provider-digitalocean  [no test files]
=== RUN   TestAccDigitalOceanKubernetesClusterNodePool_Import
--- FAIL: TestAccDigitalOceanKubernetesClusterNodePool_Import (525.76s)
    testing.go:569: Step 1 error: Resource specified by ResourceName couldn't be found: digitalocean_kubernetes_cluster_node_pool.barfoo
FAIL
FAIL    github.com/terraform-providers/terraform-provider-digitalocean/digitalocean     525.786s
FAIL
make: *** [GNUmakefile:18: testacc] Error 1
```

Any ideas on why the import failed? I am still trying to understand how non-trivial importation logic in providers is supposed to work.

Reply:
I had a typo in the resource name being checked. Fixing that in a subsequent commit now gets me this error:

```
   testing.go:569: Step 1 error: Cannot import non-existent remote object: While attempting to import an existing object to digitalocean_kubernetes_node_pool.barfoo, the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
```